Button/Button.js

```diff
@@ -43,6 +43,8 @@ const ButtonBase = kind({
 	name: 'Button',
 
 	propTypes: /** @lends agate/Button.ButtonBase.prototype */ {
+		backgroundOpacity: PropTypes.oneOf(['opaque', 'lightOpaque']),
```

Comment:
Why did you choose these values as compared to `moonstone` which used an unset value as `opaque` and set values as degrees of opacity?

Reply:
I'd say this is a case of gradually evolving preference for what my opinion is on "best" API. By using blank as the default, it means that, in the CSS, the rules must be overridden to set something else and you must override them, even if you just want to remove those rules. By setting up the API to have a default class, the CSS could be structured to work in both ways, depending on what suits the requirements, and not need to change the API later.

As for `lightOpaque`, I imagine there will likely be other levels of background opacity in the future, but this one specifically sets the background to white for some of the design's specific use cases. It could have been considered a "skin", however, it seemed like both day and night gallium designs used gray buttons as default, while occasionally needing white buttons in places. Perhaps Gallium needs a neutral or translucent variant for these cases, but that seemed premature at this point.

Reply:
I think we need to be very careful of incrementally adopting API differences like this as it runs the risk of creating divergent API semantics over time.

Reply:
We use this approach in many parts of UI and Moonstone. Button is a legacy component at this point.